topics/variant-analysis/metadata.yaml

```diff
@@ -78,6 +78,27 @@ material:
       - torhou
       - erxleben
       - bgruening
+  -
+    title: "Mapping-by-sequencing"
+    type: "tutorial"
+    name: "mapping-by-sequencing"
+    zenodo_link: "https://doi.org/10.5281/zenodo.1098033"
+    galaxy_tour: ""
+    hands_on: "yes"
```

Comment:
For `hands_on` and `slides`, can you remove the quotes around `yes` and `no`? They are booleans 😄

Reply:
@bebatut Thank you for these comments. I changed all the hands-on sections according to the preferred style.

Regarding the quotes around boolean values in the yaml file: once you're ok with the actual content of the tutorial, it may be easier to reopen this PR against the current tip of master because lots of things have changed in the yaml and dockerfiles since I created my branch.
Does that sound ok for you?

Reply:
The best would be rebase your branch against the master branch. If you want I can do it

Reply:
Well, if you have the time that would be great! Just go ahead :)